libs/platform/src/lib/smart-filter-bar/smart-filter-bar.component.html

```diff
@@ -38,7 +46,7 @@
     </fd-toolbar>
 </div>
 
-<div class="fdp-smart-filter-bar__filters" *ngIf="_showFilterBar">
+<div class="fdp-smart-filter-bar__filters" [class.fdp-smart-filter-bar__filters--hidden]="!_showFilterBar">
```

Comment:
```suggestion
<div class="fdp-smart-filter-bar__filters" [hidden]="!_showFilterBar">
```
If you do not use this class anywhere, you can use `[hidden]="!_showFilterBar"` and from the browser standpoint, it is more accessible